packages/core/lib/commands/packages.js

```diff
@@ -0,0 +1,24 @@
+var command = {
```

Comment:
We discussed separating the `truffle packages` command out into it's own pr, so that we could discuss its api in greater depth, however for the sake of getting an mvp in and across the line, I've left it in here. Happy to remove it out into it's own pr, but it's unclear right now how much more time I have available to extend the `packages` api.